src/epub/text/the-thing-on-the-doorstep.xhtml

```diff
@@ -94,7 +94,7 @@
 				<p>“I suppose you think I’m crazy, Dan⁠—but Arkham history ought to hint at things that back up what I’ve told you⁠—and what I’m going to tell you. You’ve seen one of the changes, too⁠—in your car after I told you about Asenath that day coming home from Maine. That was when she got me⁠—drove me out of my body. The last thing I remember was when I was all worked up trying to tell you <em>what that she-devil is</em>. Then she got me, and in a flash I was back at the house⁠—in the library where those damned servants had me locked up⁠—and in that cursed fiend’s body⁠ ⁠… that isn’t even human.⁠ ⁠… You know it was she you must have ridden home with⁠—that preying wolf in my body⁠—you ought to have known the difference!”</p>
 				<hr/>
 				<p>I shuddered as Derby paused. Surely, I <em>had</em> known the difference⁠—yet could I accept an explanation as insane as this? But my distracted caller was growing even wilder.</p>
-				<p>“I had to save myself⁠—I had to, Dan! She’d have got me for good at Hallowmass⁠—they hold a Sabbat up there beyond Chesuncook, and the sacrifice would have clinched things. She’d have got me for good⁠—she’d have been I, and I’d have been she⁠—forever⁠—too late⁠—My body’d have been hers for good⁠—She’d have been a man, and fully human, just as she wanted to be⁠—I suppose she’d have put me out of the way⁠—killed her own ex-body with me in it, damn her, <em>just as she did before</em>⁠—just as she, he, or it did before⁠—”</p>
+				<p>“I had to save myself⁠—I had to, Dan! She’d have got me for good at Hallowmass⁠—they hold a Sabbath up there beyond Chesuncook, and the sacrifice would have clinched things. She’d have got me for good⁠—she’d have been I, and I’d have been she⁠—forever⁠—too late⁠—My body’d have been hers for good⁠—She’d have been a man, and fully human, just as she wanted to be⁠—I suppose she’d have put me out of the way⁠—killed her own ex-body with me in it, damn her, <em>just as she did before</em>⁠—just as she, he, or it did before⁠—”</p>
```

Comment:
Incorrect change, `Sabbat` means a witch's festival, `Sabbath` means a day of rest and worship